src/_styles/_hero.scss

```diff
@@ -0,0 +1,85 @@
+.hero {
+  margin-top: $size-unit * 5; // 20px
+  font-family: "Open Sans";
+  font-style: normal;
+  font-weight: 400;
+  color: $dark-blue;
+
+  .site-description::before {
+    display: block;
+    width: $size-unit * 20; // 80px
+    border-top: $border-unit solid $dark-blue;
+    content: "";
+    margin-bottom: $size-unit * 4; // 16px
+  }
+
+  .site-description {
+    font-size: $font-size; // 14px
+    line-height: 172%;
+  }
+
+  h1 {
+    font-family: "Playfair Display";
+  }
+  .avatar {
+    display: flex;
+    justify-content: flex-end;
+
+    img {
+      width: $size-unit * 55; // 220px
+    }
+  }
+
+  .social-media {
+    position: absolute;
+    left: $size-unit * 20; // 40px + 16px + 24px
+    bottom: 0;
+  }
+
+  @media (max-width: $breakpoint-sm) {
+    .site-description::before {
+      margin-top: $size-unit * 30; // 120px
+    }
+
+    .social-media {
+      display: none;
+    }
+
+    h1 {
+      margin-top: $size-unit * 16; // 64px
+      font-size: 60px;
+      margin-bottom: $size-unit * 4; // 16px
+    }
+  }
+
+  @media (min-width: $breakpoint-sm) {
+    position: relative;
+
+    .social-media {
+      position: absolute;

```

Comment:
`position: absolute;` and `bottom: 0;` already stated above in .social-media. Make sense to remove it from here and add instead `left: unset`. This will remove `left:0;` and it will not affect to the code.

Reply:
- [ ]Done. [1373583](https://github.com/AnastasiSo/Portfolio-Project/pull/11/commits/137358394ca71c399d1f28803116b0afc1d9c55e)